apps/jan-api-gateway/application/app/interfaces/http/routes/v1/chat/completion_route.go

```
@@ -244,12 +266,33 @@ func (cApi *CompletionAPI) StreamCompletionResponse(reqCtx *gin.Context, apiKey
 	return nil
 }
 
+func (cApi *CompletionAPI) populateSelectionContext(reqCtx *gin.Context, userID uint, selection *infrainference.ProviderSelection) {
+	if selection == nil {
+		return
+	}
+	if selection.OrganizationID == nil {
+		if org, ok := auth.GetAdminOrganizationFromContext(reqCtx); ok && org != nil {
+			selection.OrganizationID = &org.ID
+		}
+	}
+
+	ctx := reqCtx.Request.Context()
+	projects, err := cApi.projectService.Find(ctx, project.ProjectFilter{MemberID: &userID}, nil)
+	if err != nil {
```

Why are you suppressing the error here?